lib/input/input_controller.h

```diff
@@ -42,6 +42,15 @@ struct InputController {
     static void mouse_button_callback(GLFWwindow* window, int button,
                                       int action, int mods);
 
+    static void set_active_window(GLFWwindow* window);
+    static GLFWwindow* get_active_window();
+
+    /********************************************************
```

Comment:
Everywhere else in the project the other style of multiline comments is used.
Example:
```C++
/**
 * @brief Dummy function that is used for comment style demonstration
 *
 */
void foo();
```